requirements.txt

```diff
@@ -0,0 +1,5 @@
+grpclib
```

Comment:
I think we should not dependence `grpclib` because the wechaty-puppet is an abstract class?

We will only need to import the payload schema from the `chatie-grpc`.